src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultItems.targets

```diff
@@ -1,8 +1,12 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <PropertyGroup>
+    <EnableDefaultAndroidItems Condition=" '$(EnableDefaultAndroidItems)' == '' ">$(EnableDefaultItems)</EnableDefaultAndroidItems>
```

Comment:
Question: is there a naming convention here?  It would be nice if this were consistent with iOS -- whatever that may be -- and I thus wonder if it should be the current `$(EnableDefault[Nount]Items)` or it should instead be `$(Enable[Noun]DefaultItems)`, e.g. `$(EnableAndroidDefaultItems)`.

Reply:
We're using `$(EnableDefault[Platform]Items)`: https://github.com/xamarin/xamarin-macios/blob/b8dac12704c162a61d0c273746b5e29b129dfff9/dotnet/targets/Xamarin.Shared.Sdk.DefaultItems.targets#L8-L11

That also seems to be the pattern in .NET: `$(EnableDefault*Items)`: https://github.com/dotnet/sdk/blob/32496b832d0e7702a0dca105160d97bf0c84c7ae/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets#L17-L19

Reply:
The xamarin-macios SDKs are following the same pattern: https://github.com/xamarin/xamarin-macios/blob/ab6b41848362b4faeb5f55eaa59d51933d3236d0/dotnet/targets/Xamarin.Shared.Sdk.DefaultItems.targets#L8-L11